tests/get_or_create/tests.py

```diff
@@ -484,3 +485,35 @@ def lock_wait():
         updated_person = Person.objects.get(first_name='John')
         self.assertGreater(after_update - before_start, timedelta(seconds=0.5))
         self.assertEqual(updated_person.last_name, 'NotLennon')
+
+
+class InvalidCreateArgumentsTests(TestCase):
```

Comment:
could this be SimpleTestCase? I guess the errors are raised before any queries happen.